lib/internal/fs/dir.js

```diff
@@ -33,6 +37,8 @@ class Dir {
   constructor(handle, path, options) {
     if (handle == null) throw new ERR_MISSING_ARGS('handle');
     this[kDirHandle] = handle;
+    this[kDirBufferedEntries] = [];
+    this[kDirBufferedEncoding] = '';
```

Comment:
Since we now have private members, would it not be better to implement these as private properties?

Reply:
I’d prefer to be consistent here – either use them for the entire class, or not.

(i.e. I’m not just doing this because I think private properties are the worst thing to happen to JS so far.)

Reply:
I guess we might switch to private properties for a couple of classes at some point though. In that case it's more churn to change it (and more work). But it's nothing I feel strongly about here, especially since I see the point in keeping it consistent.